patternlab-config.json

```diff
@@ -1,6 +1,6 @@
 {
   "cacheBust": true,
-  "cleanPublic": true,
+  "cleanPublic": false,
```

Comment:
for the local build test we are running incremental builds, that is the fastest way to recompile with the watch task - we likely want a "cleanPublic: true" for the default build

Reply:
I'd like to try incremental builds for performance and then revisit if/when it starts complicating things.